### ISSUE TITLE
Docker build broken

### ISSUE THREAD

**Issue by peterkaminski** [GovReady/govready-q#1010]
The Docker build is currently broken, with this error:

```
    building 'MySQLdb._mysql' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/MySQLdb
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -Dversion_info=(2,0,1,'final',0) -D__version__=2.0.1 -I/usr/include/mysql -I/usr/include/python3.6m -c MySQLdb/_mysql.c -o build/temp.linux-x86_64-3.6/MySQLdb/_mysql.o
    MySQLdb/_mysql.c:46:20: fatal error: Python.h: No such file or directory
     #include "Python.h"
                        ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```
Reviewing the [Docker Hub build logs](https://hub.docker.com/repository/docker/govready/govready-q/builds), the build of commit `4deaa551` on 4 Aug 2020 succeeded, and the next commit `d23a193a` on 6 Aug 2020 and subsequent builds have failed.

**Comment by peterkaminski** [comment]
During initial diagnosis, building with the Dockerfile from `4deaa551` and/or using the base image `FROM centos:centos7.8.2003` still had the same failure.

**Comment by peterkaminski** [comment]
Fixed in #1011.

CentOS library filename in EPEL apparently changed, the name `python36u-devel.x86_64` used to work, but now returns "No package python36u-devel.x86_64 available" on `yum install`.

Changing the name `python36u-devel` fixed the problem, and allows the extension build to find `Python.h`.
